iot/http_example/resources/README.md

```diff
@@ -0,0 +1,4 @@
+# Test public certificate files
+
+The public certificates in this folder are only provided for testing and should
+not be used for registering your devices.
```

Comment:
Do we use these in _our_ CI tests - if so, I assume they are sufficiently secure?

Reply:
Yes, these certificates are used in our CI tests. The authorization credentials are revoked when the tests tear down.